Database.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+const dateFormat = "20060102"
+
+func setupDB() (*sql.DB, error) {
+	dbPath := os.Getenv("TODO_DBFILE")
+	if dbPath == "" {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return nil, fmt.Errorf("directory error: %v", err)
+		}
+		dbPath = filepath.Join(cwd, "scheduler.db")
+	}
+	db, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return nil, fmt.Errorf("error connecting to the DB: %v", err)
+	}
+
+	if _, err := os.Stat(dbPath); os.IsNotExist(err) {
+		createTableQuery := `
+		CREATE TABLE scheduler (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			date TEXT NOT NULL,
+			title TEXT NOT NULL,
+			comment TEXT,
+			repeat TEXT(128)
+		);
+		CREATE INDEX idx_date ON scheduler(date);
+		`
+		if _, err := db.Exec(createTableQuery); err != nil {
+			db.Close()
+			return nil, fmt.Errorf("error creating the table: %v", err)
```

Comment:
При выводе в лог `%v`, при оборачивании `%w`
```suggestion
			return nil, fmt.Errorf("error creating the table: %w", err)
```